misc-utils/uuidgen.1.adoc

```diff
@@ -42,6 +42,12 @@ Use MD5 as the hash algorithm.
 *-s*, *--sha1*::
 Use SHA1 as the hash algorithm.
 
+*-6*, *--time-v6*::
+Generate a time-based UUID. This method creates a UUID based on the system clock plus and is lexicographically sortable according to the contained timestamp.
+
+*-7*, *--time-v7*::
+Generate a time-based UUID. This method creates a UUID based on the system clock plus and is lexicographically sortable according to the contained timestamp.
+
```

Comment:
It seems there is no difference between --time-v7 and --time-v6, copy & past bug?

Reply:
Indeed, I'll fix it.